index.html

```diff
@@ -1638,18 +1597,19 @@ <h2>Services</h2>
     <h1>Representations</h1>
 
     <p>
-A concrete serialization of a [=DID document=] in this specification is
-called a [=representation=]. A [=representation=] is created by
-serializing the <a href="#data-model">data model</a> through a process called
+A concrete serialization of a [=DID document=] in this specification is called a
+[=representation=]. A [=representation=] is created by serializing the <a
+href="#data-model">data model</a> through a process called
 <dfn>production</dfn>. A [=representation=] is transformed into the <a
 href="#data-model">data model</a> through a process called
 <dfn>consumption</dfn>. The <em>production</em> and <em>consumption</em>
 processes enable the conversion of information from one [=representation=] to
-another. This specification defines [=representations=] for JSON and JSON-LD,
-and developers can use any other [=representation=], such as XML or
-YAML, that is capable of expressing the <a href="#data-model">data model</a>.
-The following sections define the general rules for [=production=] and
-[=consumption=], as well as the JSON and JSON-LD [=representations=].
+another. This specification defines a [=representation=] for JSON, which is
+also compatible with processors that perform JSON-LD processing. Developers can
+use any other [=representation=], such as CBOR or YAML, that is capable of
+expressing the <a href="#data-model">data model</a>. The following sections
```

Comment:
```suggestion
also compatible with processors that perform JSON-LD processing.  The following sections
```

Single representation right?